src/library/masq.js

```diff
@@ -531,6 +549,10 @@ class Masq {
     if (!this.masterKey) throw Error('MasterKey is not set')
   }
 
+  _checkNonce () {
+    if (!this.nonce) throw Error('nonce is not set')
+  }
```

Comment:
Check if _checkNonce and _checkMK are necessary as masq-common already throws the error

Reply:
yes removed.